scripts/commands/release.lua

```diff
@@ -6,9 +6,25 @@
 cmdprops =
 {
     permission = 1,
-    parameters = ""
-};
+    parameters = "s"
+}
 
-function onTrigger(player)
-    player:release();
-end
\ No newline at end of file
+function error(player, msg)
+    player:PrintToPlayer(msg)
+    player:PrintToPlayer("!release {name}")
+end
+
+function onTrigger(player, name)
+    local target
```

Comment:
Initialize to nil, plox

Reply:
That is initialized as nil. When you don't specify thats what lua does. i fot told to stop doing = nil there for this reason.